.github/ISSUE_TEMPLATE/documentation_bug_report.md

```diff
@@ -1,35 +1,41 @@
 ---
 name: "\U0001F41B Documentation/aptos.dev Bug report"
 about: Create a bug report to help improve the Aptos Developers' Website
-title: "[Bug]"
+title: "[Docs]"
 labels: ["bug", "documentation"]
-assignees: ''
+assignees: 'clay-aptos'
 
 ---
 
-# 🐛 Bug
+# Aptos Documentation Issue
 
-<!-- A clear and concise description of your issue with the documentation or website.
-To report a security issue, please email security@aptoslabs.com. -->
+Use this template to report bugs and feature requests for the technical documentation and site found at Aptos.dev. All other requests should use the other Aptos bug templates:
```

Comment:
personally I think you should continue using the `<!--` and `-->` to surround the "instructions" otherwise this will clutter up the issue whenever someone doesn't delete the instruction text or doesn't fill out a section (which happens a lot) and therefore makes for very noisy tickets.

Reply:
Great suggestion. Thanks. Done!